src/tools/rbd_wnbd/rbd_mapping.h

```diff
@@ -116,4 +119,9 @@ class RbdMappingDispatcher
 
   int create(Config& cfg);
   std::shared_ptr<RbdMapping> get_mapping(std::string& devpath);
+  std::set<std::string> get_mapped_devpaths();
+  int get_mappings_count();
```

Comment:
Do these two methods need to be public?

Reply:
I thought we might use them outside this class, but I can make them private for now.